ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/ShowOrgBasemaps/ShowOrgBasemaps.cpp

```diff
@@ -113,20 +110,21 @@ QString ShowOrgBasemaps::mapLoadError() const
 
 void ShowOrgBasemaps::load(bool anonymous)
 {
-  if (!m_portal)
-    return;
+  m_portal.reset(new Portal(this));
+  connectLoadStatusSignal();
 
-  if (anonymous)
-    load();
-  else {
+  if (!anonymous && m_portal) {
```

Comment:
Use consistent bracket placement style in .cpp files.

Reply:
Done.